assets/scss/components/_footer.scss

```diff
@@ -45,15 +45,6 @@
   &__link {
     @include govuk-link-common;
     @include govuk-link-style-text;
-
```

Comment:
mixin no longer exists, checked that the undesired behaviour in this comment is no longer an issue